### PR TITLE
fips-compliance

### PR DESCRIPTION
**Pull request by resmaa**
**3.1 Release Feature**

FIPS Compliance

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1063#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | fcc09ae01426447e623eb21b0669ca59ab440a94 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1063/librarium_build/1606003790979272704) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/15-compliance/00-fips-compliance.md
Comment (karl-cardenas-coding) [inline content/docs/15-compliance/00-fips-compliance.md:111]:
```suggestion
### Platforms Supported:
```
Let's not limited it to clouds as we will probably have private data centers platforms in the future.

File: content/docs/15-compliance/00-fips-compliance.md
Comment (karl-cardenas-coding) [inline content/docs/15-compliance/00-fips-compliance.md:107]:
```suggestion
FIPS enablement at the tenant or project scope can be disabled or enabled during cluster creation.  Toggle the **FIPS Mode** button while creating the cluster configurations of cluster deployment.
```

File: content/docs/15-compliance/00-fips-compliance.md
Comment (karl-cardenas-coding) [inline content/docs/15-compliance/00-fips-compliance.md:105]:
```suggestion
<WarningBox>
```
This is pretty important so we should make it a warning box.

File: content/docs/15-compliance/00-fips-compliance.md
Comment (karl-cardenas-coding) [inline content/docs/15-compliance/00-fips-compliance.md:109]:
```suggestion
</WarningBox>
```

File: content/docs/15-compliance.md
Comment (karl-cardenas-coding) [inline content/docs/15-compliance.md:44]: Resma, at the end of this section, we should link to the new page you are creating.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and changes. Resma, I'm confused about the tenant and project scope. It says the infra layers must be FIPS enabled. Does that mean that someone could deploy a cluster that is not FIPS enabled even though I specified I want FIPS at the platform level?

File: content/docs/15-compliance/00-fips-compliance.md
Comment (resmaa) [reply content/docs/15-compliance/00-fips-compliance.md:95]: its the mandatory condition

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1063#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/3.1-release-notes/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/15-compliance.md
Comment (resmaa) [reply content/docs/15-compliance.md:40]: done

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1063#pullrequestreview-1229668597" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1063#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/3.1-release-notes/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
